modules/ROOT/pages/installation-prerequisites.adoc

```diff
@@ -48,3 +48,11 @@ Wait to receive the notification email that your domain is registered, and then
 . Search for *My Domain* using the *Quick Find* box and then select *My Domain*.
 . If the screen indicates your domain is ready, select *Log in*.
 . Select *Deploy to Users*.
+
+== Supported Browsers
+
+Community admins, community members, and guest users can access an ACM community with the latest versions of these browsers:
```

Comment:
```suggestion
Community admins, community members, and guest users can access an ACM community by using the latest versions of these browsers:
```
...just to make crystal clear that it is not the community that has "the latest versions...."

Reply:
That is better. Thank you!